ingress/rpc.go

```diff
@@ -44,14 +45,21 @@ type RPCServer interface {
 	SendResponse(ctx context.Context, request proto.Message, info *livekit.IngressInfo, err error) error
 	// SendUpdate sends an ingress info update
 	SendUpdate(ctx context.Context, info *livekit.IngressInfo) error
+	// GetEntityChannel returns a subscription for entity requests
+	GetEntityChannel(ctx context.Context) (utils.PubSub, error)
+}
+
+type RPC interface {
+	RPCClient
+	RPCServer
 }
 
 type RedisRPC struct {
 	nodeID livekit.NodeID
 	bus    *utils.RedisMessageBus
 }
 
-func NewRedisRPCClient(nodeID livekit.NodeID, rc *redis.Client) RPCClient {
+func NewRedisRPC(nodeID livekit.NodeID, rc *redis.Client) RPC {
```

Comment:
Let's remove NewRedisRPCServer since they're being combined

Reply:
Or, add SendRequest to RPCServer and SendResponse to RPCClient

Reply:
I initially removed it, but ended up adding it back specifically for the ingress handler program, since the current way of allocating node ids would cause it to share a node with the main service. Sharing this ID would not be a problem now, since the handler does not send requests, but would if it ever does.